package.json

```diff
@@ -72,6 +72,7 @@
     "ngimport": "^0.6.0",
     "pretty-ms": "^5.0.0",
     "prop-types": "15.6.1",
+    "puppeteer": "^2.1.0",
```

Comment:
are you sure this is necessary? I didn't have any trouble with chrome headless on GHA. I'd prefer to avoid adding another dependency if possible.

Reply:
I did not realize GHA has it's own, so you're right that this is not needed. Removed.